### PR TITLE
visual changes, fix #601

### PR DESCRIPTION
**Pull request by anton-tsyganenko**

### CODE REVIEW

File: 3rdparty/fullcalendar/css/fullcalendar.css [hunk 1]
```diff
@@ -403,6 +403,7 @@ table.fc-border-separate {
 .fc-grid .fc-day-number {
 	float: right;
 	padding: 0 2px;
+	font-size: 1.5em;
```
Comment (georgehrke) [inline 3rdparty/fullcalendar/css/fullcalendar.css]: please overwrite this rule in css/style.css and don't modify 3rdparty software

**Comment by georgehrke** [comment]
please review @owncloud/designers 
I am not sure about these changes

**Comment by jbtbnl** [comment]
@anton-tsyganenko thanks for contributing! Since your PR is targeting multiple issues, let's discuss them one by one:

**more focus on ui-dialog**
Good point to address this, however we are moving away from using outline shadows in ownCloud to establish a more clean design. As an alternative, please take a look at how the import dialog in Contacts works (master branch): it darkens the background to focus on the ui-dialog. (screenshot below)

screenshot from 2014-10-27 22 36 09

**button margin**
Good catch, the button is indeed aligned incorrectly. Your fix is to shift the button downward, but instead I'd propose to shift it left (as your [image](https://cloud.githubusercontent.com/assets/4764069/4796764/f7e5768a-5e05-11e4-9d17-394e75c3b64c.png) already shows) so that it aligns with the inputs above it.

**font size in calendar**
The font size might be too small on your screen, but you can easily fix this by zooming in using your browser settings (or Ctrl + scroll). As far as I know we try to keep different font sizes down to a minimum. @jancborchardt might elaborate on this.

**Comment by anton-tsyganenko** [comment]
@jbtbnl, If I zoom it in my browser, the sidebar will be too wide, so I think, the nubmers should be 1.5em or even 2em sized.

Do you mean, that I should decrease the button's margin?

I'm not good at JS, so I don't know, hot to darken the background, sorry. Should I remove the box-shadow?

what about changing the background of the current date?

**Comment by jancborchardt** [comment]
@anton-tsyganenko can you rebase your pull request? Also please describe the things you changed (ideally with screenshots) so it’s easier to review.

You’re also very welcome in our IRC channel #owncloud-calendar! :)

**Comment by anton-tsyganenko** [comment]
There are some screenshots [here](https://github.com/owncloud/calendar/issues/601). And how should I rebase the pull request?

**Comment by jancborchardt** [comment]
@anton-tsyganenko looking at the changes, I would like to cherry-pick only the margin fix for now: https://github.com/anton-tsyganenko/calendar/commit/99a544c636acb009bda4a68fce48cd414d126b77

Is that ok with you? Next time it’s best to open individual pull requests when the changes are so different. :)

**Comment by anton-tsyganenko** [comment]
Yes, it's ok.

**Comment by jancborchardt** [comment]
@anton-tsyganenko actually, can you open a new pull request for just that? I can’t cleanly cherry-pick because the data for it is in 2 separate commits. And I want to make sure your name shows up in the git history. ;)

**Comment by anton-tsyganenko** [comment]
I'll try. I think, you can close this PR.

**Comment by anton-tsyganenko** [comment]
@jancborchardt, the margin looks to be already fixed in Owncloud demo. :)

**Comment by jancborchardt** [comment]
@anton-tsyganenko oh, ok. :) I guess you can close this and the original issue then. :)

**Comment by DeepDiver1975** [comment]
rebased branch is here - no idea if this really is something to work on at the moment
https://github.com/owncloud/calendar/pull/950

@jancborchardt

**Comment by jancborchardt** [comment]
Closing this in favor of https://github.com/owncloud/calendar/pull/950 – I’m still against the changes. Except the margin fix, but that seems to have been fixed in master already.

**Pull request closed without merging**